SwiftParserCLI/Package.swift

```diff
@@ -0,0 +1,45 @@
+// swift-tools-version:5.7
+import PackageDescription
```

Comment:
I would add a newline between the tools version and the `import` to match the style of the top level Package.swift